src/rf/apps/core/enums.py

```diff
@@ -25,6 +27,7 @@
     (STATUS_CREATED, 'Created'),
     (STATUS_UPLOADED, 'Uploaded'),
     (STATUS_VALIDATED, 'Validated'),
+    (STATUS_INVALID, 'Invalid'),
```

Comment:
Also, I find it confusing how we have 3 different verb conjugations for these status names. Can't think of a better solution at the moment though.